system/infra-monitoring/requirements.lock

```diff
@@ -1,4 +1,4 @@
-dependencies:
+git dependencies:
```

Comment:
not sure why this change?

Reply:
Probably due to git conflicts / rebasing. Thank you for pointing this out.